src/compiler/binder.ts

```diff
@@ -1276,6 +1284,22 @@ namespace ts {
             activeLabels!.pop();
         }
 
+        function isDottedName(node: Expression): boolean {

```

Comment:
what's the difference to `isEntityNameExpression`?

Reply:
Good catch! No difference, will change to use the existing function.